src/cmd/flux-mini.py

```diff
@@ -964,8 +964,8 @@ def submit_async_with_cc(self, args, cclist=None):
             self.progress_start(args, len(cclist))
 
         for i in cclist:
-            #  substitute any {cc} in args and create jobspec:
```

Comment:
commit message for d16bdb4d2b522033bb572963ac7eb0678e672f21

Drop "...in this case' from first sentence?

s/anythin/anything/